tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Keboola\Test\Backend\ExternalBuckets;
+
+use http\Exception\RuntimeException;
```

Comment:
toto sa nepouziva

Reply:
super, jeste jsem zrusil ta TODO, protoze https://github.com/keboola/connection/pull/5079 je uz mergnute. lokalne mi to proslo na novem projektu (to je zadouci, novy projekt je potencialne nebezpecny)
![image](https://github.com/keboola/storage-api-php-client/assets/13363655/935f8f40-aee2-4d6c-b2bf-772327fc3646)